src/components/dashboard/FeedList.js

```diff
@@ -62,6 +54,15 @@ const FeedList = ({
   const feeds = data && data instanceof Array && data.length ? data : [emptyFeed]
   const flRef = createRef()
 
+  const VIEWABILITY_CONFIG = useMemo(
```

Comment:
This is because of an issue when reloading

Reply:
inline code comment explaining why